harness/run_eval.py

```
@@ -1594,18 +1545,45 @@ def _inventory_summary() -> Dict[str, Any]:
+                        rec = {
+                            "model": slug,
+                            "error": f"Item processing timed out after {item_timeout}s",
+                            "timeout": True,
+                        }
+                        with write_lock:
+                            f_combined.write(json.dumps(rec) + "\n")
+                            model_files[slug].write(json.dumps(rec) + "\n")
+                            f_combined.flush()
+                            model_files[slug].flush()
+                    except Exception:
+                        pass
```

Ruff (0.14.3)

`try`-`except`-`pass` detected, consider logging the exception

(S110)